tests/e2e/utils/resource_group_utils.go

```diff
@@ -0,0 +1,37 @@
+package utils
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2018-05-01/resources"
+	"github.com/Azure/go-autorest/autorest/to"
+
+	"k8s.io/apimachinery/pkg/util/uuid"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+// CreateTestResourceGroup create a test rg
+func CreateTestResourceGroup(tc *AzureTestClient) (*resources.Group, func(string)) {
+	gc := tc.createResourceGroupClient()
+	rgName := to.StringPtr("e2e-" + string(uuid.NewUUID())[0:4])
+	rg, err := gc.CreateOrUpdate(context.Background(), *rgName, createTestTemplate(rgName))
+	Expect(err).NotTo(HaveOccurred())
+	By(fmt.Sprintf("resource group %s created", *rgName))
+
+	return &rg, func(rgName string) {
+		Logf("cleaning up test resource group %s", rgName)
+		_, err := gc.Delete(context.Background(), rgName)
+		Expect(err).NotTo(HaveOccurred())
+		return
+	}
+}
+
+func createTestTemplate(name *string) resources.Group {
+	return resources.Group{
+		Name:     name,
+		Location: to.StringPtr("eastasia"),
```

Comment:
We shouldn't use a fixed location here. Seems the location is not part of e2e configure file, so let's get it from node label `failure-domain.beta.kubernetes.io/region`.

If there're something wrong when getting this, then use a default location `eastus2`, which is same as test-infra configs.

Reply:
Got it.